### PR TITLE
`Text`: update docs to improve accessibility info (HDS-2528)

### PR DESCRIPTION
**Pull request by andgen404**
### :pushpin: Summary

If merged, this PR adds in different vocabulary for the code examples and the accessibility tab

### :link: External links

Jira ticket: [HDS-2528](https://hashicorp.atlassian.net/browse/HDS-2528)

***

### 👀 Component checklist

- [ ] Percy was checked for any visual regression
- [ ] A11y tests have been run locally (`yarn test:a11y --filter="COMPONENT-NAME"`)
- [ ] If documenting a new component, an acceptance test that includes the `a11yAudit` has been added
- [ ] A changelog entry was added via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-2528]: https://hashicorp.atlassian.net/browse/HDS-2528?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MelSumner — APPROVED** [review]
I really like these changes, they reinforce markup concepts. Nice!

File: website/docs/components/text/partials/code/how-to-use.md [hunk 1]
```diff
@@ -73,19 +73,19 @@ Refer to the [Component API](#component-api) section to see what the allowed com
 To specify a text alignment, use the `@align` argument:
 
 ```handlebars
-<Hds::Text::Display @tag="h1" @align="right">This text is right-aligned</Hds::Text::Display>
-<Hds::Text::Body @tag="p" @align="center">This text is center-aligned</Hds::Text::Body>
-<Hds::Text::Code @tag="pre" @align="left">This text is left-aligned</Hds::Text::Code>
+<Hds::Text::Display @tag="h1" @align="right">Page title with right-alignment</Hds::Text::Display>
+<Hds::Text::Body @tag="p" @align="center">Paragraph with center-alignment</Hds::Text::Body>
+<Hds::Text::Code @tag="pre" @align="left">Code sample with left-alignment</Hds::Text::Code>
```
Comment (MelSumner) [inline website/docs/components/text/partials/code/how-to-use.md:78]: [suggestion] could also consider "Page Title, right-aligned" or "Page Title (right-aligned)"

**Review by MelSumner — APPROVED** [review]

File: website/docs/components/text/partials/code/how-to-use.md [hunk 2]
```diff
@@ -73,19 +73,19 @@ Refer to the [Component API](#component-api) section to see what the allowed com
 To specify a text alignment, use the `@align` argument:
 
 ```handlebars
-<Hds::Text::Display @tag="h1" @align="right">This text is right-aligned</Hds::Text::Display>
-<Hds::Text::Body @tag="p" @align="center">This text is center-aligned</Hds::Text::Body>
-<Hds::Text::Code @tag="pre" @align="left">This text is left-aligned</Hds::Text::Code>
+<Hds::Text::Display @tag="h1" @align="right">Page title with right-alignment</Hds::Text::Display>
```
Comment (KristinLBradley) [inline website/docs/components/text/partials/code/how-to-use.md:76]:
```suggestion
<Hds::Text::Display @tag="h1" @align="right">Page title, right-aligned</Hds::Text::Display>
```

File: website/docs/components/text/partials/code/how-to-use.md [hunk 3]
```diff
@@ -73,19 +73,19 @@ Refer to the [Component API](#component-api) section to see what the allowed com
...
+<Hds::Text::Body @tag="p" @align="center">Paragraph with center-alignment</Hds::Text::Body>
```
Comment (KristinLBradley) [inline website/docs/components/text/partials/code/how-to-use.md:77]:
```suggestion
<Hds::Text::Body @tag="p" @align="center">Paragraph, center-aligned</Hds::Text::Body>
```

File: website/docs/components/text/partials/code/how-to-use.md (hunk 1 above)
Comment (KristinLBradley) [inline website/docs/components/text/partials/code/how-to-use.md:78]:
```suggestion
<Hds::Text::Code @tag="pre" @align="left">Code sample, left-aligned</Hds::Text::Code>
```

**Review by jorytindall — APPROVED** [review]
:shipit:

**Pull request merged**